test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -22,6 +22,8 @@ export function setup() {
 
 			await app.workbench.sqlNotebook.runActiveCell();
 			await app.workbench.sqlNotebook.waitForActiveCellResults();
+
+			await app.workbench.quickaccess.runCommand('workbench.action.revertAndCloseActiveEditor');
```

Comment:
Do you not want to do this for all the tests?

Reply:
I guess all the others seem to close the editor already, but `can perform basic text cell functionality` doesn't seem to be doing that.

Reply:
Yes, I will add it for `can perform basic text cell functionality`. Thanks for catching that.